arviz/plots/kdeplot.py

```diff
@@ -74,6 +75,8 @@ def plot_kde(
     contour : bool
         If True plot the 2D KDE using contours, otherwise plot a smooth 2D KDE.
         Defaults to True.
+    hdi_probs : list
+        Confidence levels for highest density (2-dimensional) interval contours of a 2D KDE.
```

Comment:
Confidence feels strange, in this context, maybe something like "Plots highest density credibility regions for the provided probabilities. Defaults to matplotlib chosen levels with no fixed probability associated"?

Reply:
I've updated this in 6f2d912